src/Sylius/Behat/Context/Setup/ChannelContext.php

```diff
@@ -114,7 +115,8 @@ public function storeOperatesOnASingleChannel($currencyCode = null)
      */
     public function theStoreOperatesOnAChannelNamed($channelIdentifier, $currencyCode = null)
     {
-        $defaultData = $this->defaultChannelFactory->create($channelIdentifier, $channelIdentifier, $currencyCode);
+        $channelCode = StringInflector::nameToLowercaseCode($channelIdentifier);
```

Comment:
It is not used anywhere, so I would put it in one line. But that's just a suggestion

Reply:
I'm doing it like this because IMHO it's more readable. Shall I change it?

Reply:
Not needed. Just a suggestion. It is not something I would kill for 🔪

Reply:
Since it has introduced `$channelCode`, I would rename `$channelIdentifier` to `$channelName`